### PR TITLE
refac: Clean up contracts

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## Description

Remove contracts that are no longer used.
Update contracts to be closer to the desired state.
Remove old bad tests and test file (time series json file).

TODO
- Consider whether we want to change the name of the observation time columns in the `time_series_point_schema`

## References

* #610

### CODE REVIEW

File: source/contracts/metering-point-domain/metering-point-connection-state.json [hunk 1]
```diff
@@ -1,13 +1,5 @@
 {
   "literals": [
```
Comment (lasrinnil) [inline source/contracts/metering-point-domain/metering-point-connection-state.json:2]: Is the `ConnectionState` still relevant for the wholesale domain? this stil relevant

File: source/contracts/time-series-domain/time-series-resolution.json [hunk 2]
```diff
@@ -1,24 +1,12 @@
 {
   "literals": [
-    {
-      "name": "unknown",
-      "value": 0
-    },
     {
       "name": "quarter",
```
Comment (lasrinnil) [inline source/contracts/time-series-domain/time-series-resolution.json:4]: Perhaps I am missing something, but the contract for the time series does not have a resolution.

https://github.com/Energinet-DataHub/opengeh-wholesale/blob/main/docs/inter-domain-integration/time-series-point-schema.py

File: source/databricks/package/schemas/time_series_point_schema.py [hunk 3]
```diff
@@ -35,26 +35,20 @@
...
-
+        StructField("Quantity", DecimalType(18, 6), True),
         # "A02" (missing) | "A03" (estimated) | "A04" (measured) | "A06" (calculated)
         # Example: A02
         StructField("Quality", StringType(), False),
-        
         # The time when the energy was consumed/produced/exchanged
-        StructField("ObservationTime", TimestampType(), False),
-        
+        StructField("Time", TimestampType(), False),
```
Comment (lasrinnil) [inline source/databricks/package/schemas/time_series_point_schema.py:46]: Is this changed in alignment with Volt?

Comment (BjarkeMeier) [reply source/databricks/package/schemas/time_series_point_schema.py:46]: My personal preference would be to revert this change in an upcoming change. The reason is that this change doesn't currently break anything and reverting it requires splitting all usages of `Colname.time` (and year/month/day) into use cases for metering points and charge links. What are your thoughts?

This is btw the TODO issue in the PR description.

File: source/contracts/time-series-domain/time-series-resolution.json (hunk 2 above)
Comment (BjarkeMeier) [reply source/contracts/time-series-domain/time-series-resolution.json:4]: Good catch. Thx.

File: source/contracts/metering-point-domain/metering-point-connection-state.json (hunk 1 above)
Comment (BjarkeMeier) [reply source/contracts/metering-point-domain/metering-point-connection-state.json:2]: No, it's not. Good catch.

File: source/databricks/package/schemas/time_series_point_schema.py (hunk 3 above)
Comment (lasrinnil) [reply source/databricks/package/schemas/time_series_point_schema.py:46]: Ah now I am up to speed :) 

Let's keep the change for now and revert it when needed.

**Review by lasrinnil — APPROVED** [review]

File: source/databricks/package/schemas/time_series_point_schema.py (hunk 3 above)
Comment (BjarkeMeier) [reply source/databricks/package/schemas/time_series_point_schema.py:46]: I'd like to do it now as we have now introduced a diversion from the version agreed with team Volt.

**Pull request merged**